pkg/webhook/resourceusage/util.go

```diff
@@ -112,7 +112,7 @@ func parseJavaMemoryString(str string) (int64, error) {
 }
 
 // Logic copied from https://github.com/apache/spark/blob/c4bbfd177b4e7cb46f47b39df9fd71d2d9a12c6d/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStep.scala
-func memoryRequiredForSparkPod(spec so.SparkPodSpec, memoryOverheadFactor *string, appType so.SparkApplicationType, replicas int64) (int64, error) {
+func MemoryRequiredForSparkPod(spec so.SparkPodSpec, memoryOverheadFactor *string, appType so.SparkApplicationType, replicas int64) (int64, error) {
```

Comment:
why capitalize?

Reply:
in go its the only way you can "expose a function" outside of its class lol 
https://stackoverflow.com/questions/18013529/expose-a-function-in-go-package

Reply:
Ohh wow, had forgotten